Makefile.release

```diff
@@ -0,0 +1,6 @@
+VERSION ?= latest
```

Comment:
I think it would be helpful to have a comment in this file and in `Makefile.docker` explaining what they're for, so people can reason about why a target would be put in one or the other, and why some of the subprojects include one but not the other. I say this as someone who only understands `make` in the most superficial way; really, this documentation is for me :-)

Reply:
Also, when I run `make release VERSION=0.0.1` the `kubernetes.yaml` in the tar file reference the `latest` docker images. That won't not be stable in the long run, if the docker images evolve in an incompatible way.

Reply:
> Also, when I run make release VERSION=0.0.1 the kubernetes.yaml in the tar file reference the latest docker images. That won't not be stable in the long run, if the docker images evolve in an incompatible way.

Good point, I almost forgot about it. It is currently done through rather ugly `sed` commands. But it should do for now unless we want to first implement #31.

Reply:
>  I say this as someone who only understands make in the most superficial way; really, this documentation is for me :-)

Welcome to the club, I'm learning make right now :-D. I added some comments to `Makefile.docker` and `Makefile.release`. Please have a look if they are enough.

Reply:
Those comments are great, thanks.